rr_computer_vision/include/traffic_light.hpp

```diff
@@ -1,25 +1,53 @@
 /**
  * @file rr_traffic_light.hpp
  * @brief Traffic Light Header File
- * @author Jamie Kim
- * @author Jason Leung
- * @author Toni Ogunmade(oluwatoni)
- * @author Adrian Malaran
- * @competition IARRC 2018
+ * @author Yuchi(Allan) Zhao
+ * @competition IARRC 2019
  */
 
-#ifndef TRAFFIC_LIGHT
-#define TRAFFIC_LIGHT
+#ifndef RR_TRAFFIC_LIGHT
+#define RR_TRAFFIC_LIGHT
 
 // ROS includes
 #include <ros/ros.h>
+#include <image_transport/image_transport.h>
+#include <cv_bridge/cv_bridge.h>
+#include <sensor_msgs/image_encodings.h>
+#include <ros/console.h>
+#include <std_msgs/Int8.h>
+#include <std_msgs/Float32.h>
+#include <std_srvs/Empty.h>
+#include <std_srvs/Trigger.h>
```

Comment:
You don't actually need a lot of these includes in the header file. Only keep the includes that you need for the header to compile! But the others into the .cpp file. You can usually sanity check this quickly by just seeing if the object in the header is referenced anywhere else in the file, and it's not the case for most of these. Remove all the ones you don't need and make sure it compiles